chromium_src/chrome/browser/ui/views/tab_search_bubble_host.cc

```diff
@@ -6,19 +6,7 @@
 #include "build/build_config.h"
 #include "ui/base/ui_base_features.h"
```

Comment:
these includes doesn't appear to be relevant anymore

Reply:
Fixed in e22f77b475a96dee1eb41e6e4aeca2ddfdeebc65